tasks/install.yml

```diff
@@ -19,6 +19,7 @@
   group:
     name: ssmtp
     state: present
+  when: "{{ ssmtp.restrict == true }}"
```

Comment:
When clauses should be bare as far as I know eg.

```YAML
when: ssmtp.restrict == true
```

In fact you don't need the equality test here:

```YAML
when: ssmtp.restrict
```

Reply:
indeed, thanks for showing me how todo it correctly